src/mpi/init/get_function_table.c

```diff
@@ -0,0 +1,36 @@
+
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpiimpl.h"
+
+/*@
+   QMPIX_Get_function_table - Gets the QMPI function table to allowt the user to determine the next
+                              function or functions to call.
+
+Output Parameters:
+. funcs - A struct of functions that includes every function for every attached QMPI tool, in
+addition to the MPI library itself.
+
+   Notes:
+
+.N Fortran
+
+.N Errors
+.N MPI_SUCCESS
+@*/
+QMPI_Function_pointers_t **QMPIX_Get_function_table(void)
```

Comment:
@pavanbalaji made the point that this function might not be necessary. We could provide the pointer to the function table when calling the registration function at init time.